include/gatekeeper_lua_lpm.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * Gatekeeper - DoS protection system.
+ * Copyright (C) 2016 Digirati LTDA.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef _GATEKEEPER_LUA_LPM_H_
+#define _GATEKEEPER_LUA_LPM_H_
+
+#include <lualib.h>
+
+int l_init_lpm(lua_State *l);
+int l_lpm_add(lua_State *l);
+int l_lpm_del(lua_State *l);
+int l_lpm_lookup(lua_State *l);
+
+int l_init_lpm6(lua_State *l);
+int l_lpm6_add(lua_State *l);
+int l_lpm6_del(lua_State *l);
+int l_lpm6_lookup(lua_State *l);
```

Comment:
Instead of exporting all these functions, do the following:
1. make all these functions `static` in file `lua_lpm.c`;
2. Move `const struct luaL_reg lpmlib_lua_c_funcs` to file `lua_lpm.c` AND make it `static`;
3. Define and only export `void lualpm_openlib(lua_State *l)` to call `luaL_register(L, "lpmlib", lpmlib_lua_c_funcs);`.